src/pdf-viewer.scss

```diff
@@ -0,0 +1,34 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-pdf-viewer;
+
+.#{$block} {
+  padding: 1rem;
+
+  &__dialog {
+    max-height: calc(100vh - 4rem);
+  }
+
+  &__title {
+    background-color: transparent;
+    margin-bottom: 1rem;
+    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_BorderColor);
+    box-shadow: none;
+  }
+
+  &__content {
+    width: 100%;
+    min-height: 28rem;
+
+    &--in-page {
+      width: 100%;
+      min-height: 35rem;
```

Comment:
I got not clue where `min-height` values are coming from, I think it should be better to specify them in inline styling, instead of putting them into library.

Reply:
They've been removed.